Sources/MapboxMaps/Gestures/GestureHandlers/QuickZoomGestureHandler.swift

```diff
@@ -33,7 +35,8 @@ internal final class QuickZoomGestureHandler: GestureHandler {
             let distance = location.y - initialLocation.y
             // change by 1 zoom level per 75 points of translation
             let newZoom = initialZoom + distance / 75
-            mapboxMap.setCamera(to: CameraOptions(anchor: initialLocation, zoom: newZoom))
+            let anchor = focalPoint ?? initialLocation
+            mapboxMap.setCamera(to: CameraOptions(anchor: anchor, zoom: newZoom))
```

Comment:
There's a pattern in these gesture handlers where we capture configuration options when the gesture starts so that if the options change during the gesture, the changes won't apply until the next gesture. Let's follow that pattern for focalPoint as well. See `initialRotateEnabled` in the pinch gesture handler for an example.